code/datums/uplink/martial_arts.dm

```diff
@@ -21,3 +21,27 @@
 	desc = "A manual containing basic Gun-Kata instruction and techniques. This is for projectile weapons, allowing you to reload by using a gun on a magazine, and automatically ejecting the magazine when it's empty."
 	telecrystal_cost = 1
 	path = /obj/item/martial_manual/gun_kata
+
+/datum/uplink_item/item/martial_arts/skrell
```

Comment:
You likely want to validate that whoever is buying this martial arts can then actually use the item, since they are specie restricted below

Reply:
We have quite a few species-restricted uplink items that you can still purchase if not usable, so I figured it was better to just go with what's done elsewhere and add a warning that it can only be used by x species